act/virtualNetwork.tf

```diff
@@ -17,65 +17,65 @@ resource "azurerm_resource_group" "resource_group" {
 }
 
 resource "azurerm_virtual_network" "example" {
-  name = "vn-test"
-  address_space = ["10.0.0.0/16"]
-  location = "East US"
+  name                = "vn-test"
+  address_space       = ["10.0.0.0/16"]
+  location            = "East US"
   resource_group_name = "testrg"
 }
 
 resource "azurerm_subnet" "example" {
-  name = "subnet-test"
-  resource_group_name = "testrg"
+  name                 = "subnet-test"
+  resource_group_name  = "testrg"
   virtual_network_name = azurerm_virtual_network.example.name
-  address_prefixes = ["10.0.0.0/24"]
+  address_prefixes     = ["10.0.0.0/24"]
 }
 
 resource "azurerm_network_interface" "ni_linux" {
-  name = "linux-test"
-  location = "East US"
+  name                = "linux-test"
+  location            = "East US"
   resource_group_name = "testrg"
   ip_configuration {
-    name = "internal"
-    subnet_id = azurerm_subnet.example.id
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
     private_ip_address_allocation = "Dynamic"
   }
 }
 
 resource "azurerm_network_interface" "ni_win" {
-  name = "win-test"
-  location = "East US"
+  name                = "win-test"
+  location            = "East US"
   resource_group_name = "testrg"
   ip_configuration {
-    name = "internal"
-    subnet_id = azurerm_subnet.example.id
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.example.id
     private_ip_address_allocation = "Dynamic"
   }
 }
 
 resource azurerm_network_security_group "bad_sg" {
-  location = "East US"
-  name = "test-nsg"
+  location            = "East US"
+  name                = "test-nsg"
   resource_group_name = "testrg"
   security_rule {
-    access = "Allow"
-    direction = "Inbound"
-    name = "AllowSSH"
-    priority = 200
-    protocol = "TCP"
-    source_address_prefix = "*"
-    source_port_range = "*"
-    destination_port_range = "22-22"
+    access                     = "Allow"
+    direction                  = "Inbound"
+    name                       = "AllowSSH"
+    priority                   = 200
+    protocol                   = "TCP"
+    source_address_prefix      = "*"
+    source_port_range          = "*"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure read, write and delete request logging is enabled for queue service in Azure Storage Account**
 
**Vulnerability**
Azure Storage Account logging has not been enabled for Queue service for read, write and delete requests, this may make audit challenging.
 
**Remediation**
Storage Logging takes place server-side recording details in the storage account for both successful and failed requests. These logs allow users to see the details of read, write, and delete operations against the queues. Storage Logging log entries contain the following information about individual requests: timing information, for example start time, end-to-end latency, server latency, authentication details, concurrency information, and the size of request and response messages.

source: accurics-beta